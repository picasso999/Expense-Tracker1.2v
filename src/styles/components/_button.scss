.button{
    background: $blue;
    border: none;
    color: white;
    display: inline-block;
    font-size: $font-size-large;
    font-weight: 300;
    line-height: 1;
    padding: $s-size;
    text-decoration: none;
}

.button__logout {
    background: $no-color;
    border: none;
    color: white;
    display: inline-block;
    font-size: $font-size-large;
    font-weight: 300;
    line-height: 1;
    padding: $s-size;
    text-decoration: none;
}

.button--secondary{
    background: #888;
}