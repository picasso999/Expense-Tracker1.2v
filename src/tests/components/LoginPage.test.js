import React from 'react';
import {shallow} from 'enzyme';
import {LoginPage} from '../../components/LoginPage';

test('should correctly render loginPage',()=>{
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
});

it('should call startLogin on button click', () => {
    const startLogin = jest.fn();
    const wrapper = shallow(<LoginPage startLogin={startLogin} />);
    wrapper.find('button').simulate('click');
    expect(startLogin).toHaveBeenCalled();
  });