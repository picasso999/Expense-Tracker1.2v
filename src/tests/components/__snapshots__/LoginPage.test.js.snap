// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render loginPage 1`] = `
<div>
  <button>
    LogIn
  </button>
</div>
`;
